float32 CoatSpeed
string CoatName
int32 IndexDelay
PathStamped[] path
